cranelift/reader/src/run_command.rs

```diff
@@ -45,6 +45,7 @@ impl RunCommand {
                 };
                 if !matched {
                     let actual = DisplayDataValues(&actual);
+
```

Comment:
There are several unrelated changes across this PR. This is one of them. The change to tests/spec_testsuite is another. Could you please revert them?